src/main/java/uk/gov/hmcts/cmc/claimstore/services/ccd/callbacks/ResetRpaCallbackHandler.java

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.ccd.domain.CaseEvent;
+import uk.gov.hmcts.cmc.ccd.mapper.CaseMapper;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.Role;
+import uk.gov.hmcts.cmc.claimstore.services.ccd.callbacks.roboticssupport.RoboticsNotificationService;
+import uk.gov.hmcts.cmc.claimstore.utils.CaseDetailsConverter;
+import uk.gov.hmcts.cmc.domain.exceptions.BadRequestException;
+import uk.gov.hmcts.cmc.domain.models.Claim;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.cmc.claimstore.services.ccd.Role.CASEWORKER;
+
+@Service
+public class ResetRpaCallbackHandler extends CallbackHandler {
+    private static final String RPA_EVENT_TYPE = "RPAEventType";
```

Comment:
RPAEventTypes should be an enum CaseEvent as an example

Reply:
RPAEventType is a enum, this is just a key used to fetch the object from the map

Reply:
Sorry, I meant to refer to the constants bellow this line, i.e the ones that are used in the switch statement

Reply:
ok got it..changed now